restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -87,15 +88,15 @@
 	"dialStepNumberSendDigitsTooltip": "Enviar estos tonos DTMF después de marcar el número",
 	"dialStepNumberSendDigitsLabel": "Enviar dígitos",
 	"dialStepNumberSendDigitsInputPlaceholder": "wwww1234",
-	"dialStepNumberBeforeConnectTooltip": "'Play' o 'Say' información al destinatario, o dar opción de rechazar la llamada telefónica usando las acciones 'Collect' y 'Hangup'",
+	"dialStepNumberBeforeConnectTooltip": "'Play' o 'Say' para información al destinatario, o dar opción de rechazar la llamada telefónica utilizando las acciones 'Collect' y 'Hangup'",
 	"dialStepNumberBeforeConnectLabel": "Antes de conectar",
 	"dialStepNumberChooseTargetLabel": "Seleccione un destino",
 	"dialStepClientBeforeConnectLabel": "Antes de conectar",
 	"dialStepClientDestinationInputPlaceholder": "thomas",
 	"dialStepClientDestinationInputTooltip": "Marcar a cliente",
 	"dialStepConferenceInputPlaceholder": "Sala de conferencias",
 	"dialStepConferenceInputTooltip": "Marque para iniciar sesión",
-	"dialStepConferenceWaitTooltip": "Un módulo para ejecutar mientras espera ser añadido a la sesión. Usar el 'Play' o 'Say'",
+	"dialStepConferenceWaitTooltip": "Un módulo para ejecutar mientras espera ser añadido a la sesión. Utilizar el 'Play' o 'Say'",

```

Comment:
Grammar error found. Change for:
"dialStepConferenceWaitTooltip": "Un módulo para ejecutar mientras espera ser añadido a la sesión. Utilizar 'Play' o 'Say'",